rcttc-queries.sql

```diff
@@ -59,20 +59,24 @@ group by c.first_name;
 
 -- Calculate the total revenue per show based on tickets sold.
 
-select name, sum(ticket_price)
-from performance
-group by name;
+select p.name, p.date, p.ticket_price * count(t.ticket_id) as total_revenue
+from performance p
+inner join ticket t on p.performance_id = t.performance_id
+group by p.performance_id;
```

Comment:
We want total revenue per _show_, not per _performance_, so this can be done like this (also simplifying count of ticket multiplication by ticket price):
```
select p.name, sum(p.ticket_price)
from ticket t
inner join performance p on p.performance_id = t.performance_id
group by p.name;
```

Reply:
Your answer looks very close to what I had before, look like I just had to select from the ticket table and inner join with performance table. I definitely over complicated them when you said to take tickets into account. Thank you for your help Irina!